packages/zoe/src/zoeService/utils.ts

```diff
@@ -118,13 +104,14 @@ export type StartInstance = <SF>(
   issuerKeywordRecord?: Record<Keyword, Issuer<any>>,
   // 'brands' and 'issuers' need not be passed in; Zoe provides them as StandardTerms
   terms?: Omit<StartParams<SF>['terms'], 'brands' | 'issuers'>,
+  // @ts-expect-error xxx types puzzle
   privateArgs?: Parameters<SF>[1],
   label?: string,
 ) => Promise<StartedInstanceKit<SF>>;
 
 export type GetPublicFacet = <SF>(
-  instance: Instance<SF> | PromiseLike<Instance<SF>>,
-) => Promise<StartResult<SF>['publicFacet']>;
+  instance: Instance<SF> | PromiseLike<Instance<SF>> | unknown,
```

Comment:
`T | unknown` is `unknown` for any `T`, right? Does this render the `Instance<SF>` constraint useless?

Reply:
thanks for catching that. I gave up on trying to get that one to work. Digging into it more, I think there's a deeper problem: de27da76d4e9a70a9ce0a82818872981825cca7b

I'm removing the Zoe changes from this PR